java/system-test/src/main/java/sleeper/systemtest/SystemTestProperty.java

```diff
@@ -50,6 +52,9 @@ public interface SystemTestProperty extends InstanceProperty {
                     "number of bulk import jobs that each import all of those files.")
             .defaultValue("1")
             .validationPredicate(Utils::isPositiveInteger).build();
+    SystemTestProperty BULK_IMPORT_QUEUE_PROPERTY = Index.propertyBuilder("sleeper.systemtest.bulkimport.queue.property")
+            .description("The property for the bulk import queue which jobs should be sent to in SendBulkImportJobs.")
+            .defaultValue(BULK_IMPORT_EMR_JOB_QUEUE_URL.getPropertyName()).build();
```

Comment:
It's a bit strange how this property is going to be pointing to a property, e.g:
```properties
sleeper.systemtest.bulkimport.queue.property=sleeper.bulk.import.emr.job.queue.url
```

Perhaps we can use an option from EMR, Persistent EMR, and EKS, then figure out which property to use in SendBulkImportJobs?

Reply:
We could, although this way we won't need to update it if we add other queues. I feel like it's probably worth it to avoid needing to change it again?

Reply:
Yea true, i just thought it looked a bit odd since we havent done that before. I'm happy keeping it as is.